pkgs/development/interpreters/python/python-packages-base.nix

```diff
@@ -98,8 +98,4 @@ in {
   python = toPythonModule python;
   # Dont take pythonPackages from "global" pkgs scope to avoid mixing python versions
   pythonPackages = self;
-
-  # Remove?
-  recursivePthLoader = toPythonModule (callPackage ../../../development/python-modules/recursive-pth-loader { });
```

Comment:
For the record, I don't understand the consequences of moving this.

Reply:
I don't understand why it was there in the first place...

Reply:
Ok, well, seems reasonable to try to move it and see what happens. Either it's safe and good, or we learn something =)